### DISCUSSION TITLE
Commercial license cost

### DISCUSSION THREAD

**Discussion by bacanoo-io** [micro/go-micro#2080] [General]
Hi,
I need information about commercial license costs.
How it works.
Please give me a detailed information.

Best regards

**Comment by asim** [comment]
Still working on it. Likely $25/device.

**Reply by bacanoo-io** [reply]
This means developer - developer environment or hosted nodes?

**Comment by asim** [comment]
running instances of nitro

**Reply by bacanoo-io** [reply]
Ok, thanks for the information.
A developer license would be more considered.

**Reply by asim** [reply]
Because it would be cheaper?
